### PR TITLE
Release/0.5.0

### PR DESCRIPTION
**Pull request by domfournier**
Official release.

### CODE REVIEW

File: docs/content/release_notes.rst [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+**New Application** - Octree Mesh Creation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+New application for the creation of octree meshes with arbitrary refinements around entities.
+
+Application Updates
+^^^^^^^^^^^^^^^^^^^
+
+- Major update to the Peak Finder algorithm and application. New selection panel for the query of channel groups. Faster plotting.
+- Implementation of the ui.json for Peak Finder and Octree Mesh Creation. Parameters can be re-loaded in the
```
Comment (benk-mira) [inline docs/content/release_notes.rst:17]: Should we have a note about running from GA?

**Review by benk-mira — COMMENTED** [review]
Scan through the code changes looks good.  Will dive into the docs next to make sure they're happy and test a couple of the apps.

File: docs/content/release_notes.rst (hunk 1 above)
Comment (domfournier) [reply docs/content/release_notes.rst:17]: I thought of keeping it on the down-low until 3.3.1 is released?

**Comment by benk-mira** [comment]
image
Is there supposed to be an image of the output here?

File: docs/content/release_notes.rst (hunk 1 above)
Comment (benk-mira) [reply docs/content/release_notes.rst:17]: Yep.  Fair enough - let's iron out kinks first!

**Comment by benk-mira** [comment]
image
[Errno 2] No such file or directory: 'C:\\Users\\dominiquef\\Documents\\GIT\\mira\\geoapps\\assets\\Workspace/Gravity_Magnetics_drape60m_Airborne_TMI_RGB.txt'
I fixed up some spelling mistakes a rebuilt but I'm missing a file here.  Should we track this file in git so that both of us can build?

File: docs/content/release_notes.rst (hunk 1 above)
Comment (benk-mira) [reply docs/content/release_notes.rst:17]: Octree mesh app discusses drag and drop for 3.3.1 - was this intended?  Are we just keeping the peakfinder ui.json drag and drop on the dl?

**Comment by benk-mira** [comment]
create_surface app seems to be gone... Maybe something I did?  

... found an error logged during build:

image

**Comment by benk-mira** [comment]
Missing
image
image

**Comment by benk-mira** [comment]
Here too

image

**Comment by benk-mira** [comment]
Some kind of broken reference here (clustering):
image

**Comment by benk-mira** [comment]
A few blank axes in the clustering page eg:
image

**Comment by benk-mira** [comment]
Inversion docs talking about adding contours - is this a copy/paste error??  
image

**Comment by benk-mira** [comment]
I don't remember this being a part of pf_inversion.py is this outdated??
image

**Comment by domfournier** [comment]
You'll need to downgrade plotly like what's in the environment.yml

**Comment by benk-mira** [comment]
Should there be a rendering of the widget below the description here?
image

**Comment by benk-mira** [comment]
Downgraded sphinx and plotly to environment.yaml versions and rebuilt.  Plots are now rendering - I've removed the comments that were fixed by the downgrade.

File: docs/content/release_notes.rst (hunk 1 above)
Comment (domfournier) [reply docs/content/release_notes.rst:17]: Ok fair enough. I will add a note.

**Comment by domfournier** [comment]
> image
> [Errno 2] No such file or directory: 'C:\Users\dominiquef\Documents\GIT\mira\geoapps\assets\Workspace/Gravity_Magnetics_drape60m_Airborne_TMI_RGB.txt'
> I fixed up some spelling mistakes a rebuilt but I'm missing a file here. Should we track this file in git so that both of us can build?

Done

**Comment by benk-mira** [comment]
colorbar labels borked in create_surface application doc:
image

**Comment by benk-mira** [comment]
Still getting this one in peak_finder after running the notebook app locally:
image

**Comment by benk-mira** [comment]
What's with the missing pixel??:
image

**Comment by domfournier** [comment]
> What's with the missing pixel??:
> image

good eye. I was testing missing values. Guess I need to re-build with original FlinFlon file...

**Pull request merged**